.github/workflows/push.yml

```diff
@@ -65,7 +65,9 @@ jobs:
       - name: Verify no lint disabled in the new code
         run: |
           NEW_CODE=$(git diff origin/main..$(git branch --show-current) | grep -e '^+')
+          echo "Code diff:\n$NEW_CODE"
```

Comment:
Using '\n' within `echo` is a bit of a smell; `echo` is a shell builtin and whether this works or not depends on the shell.

Preferable alternatives that can be validated by inspection:
 - ```sh
   echo "Code diff:"
   echo "${NEW_CODE}"
   ```
 - ```sh
   printf 'Code diff:\n%s\n' "${NEW_CODE}"
   ```